### PR TITLE
add the version information of LibAFL in the UI

### PR DESCRIPTION
**Pull request by ToSeven**
The status bar will contain the name of fuzzer and the version information of LibAFL.

### CODE REVIEW

File: libafl/src/monitors/tui/ui.rs [hunk 1]
```diff
@@ -32,9 +33,10 @@ pub struct TuiUI {
 }
 
 impl TuiUI {
-    pub fn new(title: String, enhanced_graphics: bool) -> Self {
+    pub fn new(title: String, version: String, enhanced_graphics: bool) -> Self {
```
[the remark this answers is not in the document]
Comment (ToSeven) [reply libafl/src/monitors/tui/ui.rs:36]: Do you mean developers decide whether the version information is shown in the UI?  If yes, I think AFL and AFL++ both display the version information in the UI by default, so LibAFL should too. It is just my opinion.

[the remark this answers is not in the document]
Comment (tokatoka) [reply libafl/src/monitors/tui/ui.rs:36]: but libafl is a library
shouldn't ui display the version of the actual fuzzer?

[the remark this answers is not in the document]
Comment (tokatoka) [reply libafl/src/monitors/tui/ui.rs:36]: so it's better to expose an API with which users can set any version str

[the remark this answers is not in the document]
Comment (ToSeven) [reply libafl/src/monitors/tui/ui.rs:36]: Well, I will commit a new PR containing the `with_version" API

File: libafl/src/monitors/tui/mod.rs [hunk 2]
```diff
@@ -362,7 +352,10 @@ fn run_tui_thread(
 
         let backend = CrosstermBackend::new(stdout);
         let mut terminal = Terminal::new(backend)?;
-        let mut ui = TuiUI::new(title, enhanced_graphics);
+
+        let version = env!("CARGO_PKG_VERSION").to_string();
```
Comment (tokatoka) [inline libafl/src/monitors/tui/mod.rs:356]: this change is not needed?

Comment (ToSeven) [reply libafl/src/monitors/tui/mod.rs:356]: I forget to delete this line.

File: fuzzers/nyx_libxml2_standalone/src/main.rs [hunk 3]
```diff
@@ -10,6 +10,7 @@ use libafl::{
     feedbacks::{CrashFeedback, MaxMapFeedback},
     inputs::BytesInput,
     monitors::tui::TuiMonitor,
+    monitors::tui::ui::TuiUI,
```
Comment (tokatoka) [inline fuzzers/nyx_libxml2_standalone/src/main.rs:13]: can you run cargo fmt

**Comment by tokatoka** [comment]
wait.. can't we just put it in the status bar?

File: fuzzers/nyx_libxml2_standalone/src/main.rs (hunk 3 above)
Comment (ToSeven) [reply fuzzers/nyx_libxml2_standalone/src/main.rs:13]: Yes, I have run `cargo fmt` before I commit this PR.

Comment (tokatoka) [reply fuzzers/nyx_libxml2_standalone/src/main.rs:13]: You need to run it both in the project's top directory and the directory of the fuzzer that you changed

**Pull request merged**